botocore-stubs/awsrequest.pyi

```diff
@@ -58,7 +59,7 @@ class AWSPreparedRequest:
         method: str,
         url: str,
         headers: HTTPHeaders,
-        body: str,
+        body: Union[str, bytes, bytearray, io.IOBase, None],
```

Comment:
I'm not 100% on the use of io.IOBase here. It probably wouldn't be unreasonable to go to Any here, honestly.

Reply:
`IOBase` looks reasonable. I am going to verify it.

Reply:
What do you think of?

```python
from typing import IO

body: Union[bytes, str, bytearray, IO[bytes], IO[str], None]
```

Reply:
Makes sense to me. I knew there was a variant of IO somewhere that I was missing.